src/Icons/src/DependencyInjection/UXIconsExtension.php

```diff
@@ -42,32 +42,66 @@ public function getConfigTreeBuilder(): TreeBuilder
                 ->variableNode('default_icon_attributes')
                     ->info('Default attributes to add to all icons.')
                     ->defaultValue(['fill' => 'currentColor'])
+                    ->example(['class' => 'icon'])
+                ->end()
+                ->arrayNode('icon_sets')
+                    ->info('Icon sets configuration.')
+                    ->defaultValue([])
+                    ->normalizeKeys(false)
+                    ->useAttributeAsKey('prefix')
+                    ->arrayPrototype()
+                        ->info('the icon set prefix (e.g. "acme")')
+                        ->children()
+                            ->scalarNode('path')
+                                ->info("The local icon set directory path.\n(cannot be used with 'alias')")
+                                ->example('%kernel.project_dir%/assets/svg/acme')
+                            ->end()
+                            ->scalarNode('alias')
+                                ->info("The remote icon set identifier.\n(cannot be used with 'path')")
+                                ->example('simple-icons')
+                            ->end()
+                            ->arrayNode('icon_attributes')
+                                ->info('Override default icon attributes for icons in this set.')
+                                ->example(['class' => 'icon icon-acme', 'fill' => 'none'])
+                                ->normalizeKeys(false)
+                                ->variablePrototype()
+                                ->end()
+                            ->end()
+                        ->end()
+                    ->end()
+                    ->validate()
+                        ->ifTrue(fn (array $v) => isset($v['path']) && isset($v['alias']))
+                        ->thenInvalid('You cannot define both "path" and "alias" for an icon set.')
+                    ->end()
                 ->end()
                 ->arrayNode('aliases')
-                    ->info('Icon aliases (alias => icon name).')
-                    ->example(['dots' => 'clarity:ellipsis-horizontal-line'])
+                    ->info('Icon aliases (map of alias => full name).')
+                    ->example([
+                        'dots' => 'clarity:ellipsis-horizontal-line',
+                        'privacy' => 'bi:cookie',
+                    ])
                     ->normalizeKeys(false)
                     ->scalarPrototype()
                         ->cannotBeEmpty()
                     ->end()
                 ->end()
                 ->arrayNode('iconify')
-                    ->info('Configuration for the "on demand" icons powered by Iconify.design.')
+                    ->info('Configuration for the remote icon service.')
```

Comment:
Why not keeping reference to Iconify?

Reply:
The key is iconify, and the name and website are given just after ;) 

But as we start to more and more use the term "remote icon" and "local icon" i wanted to add it here